Gemfile.lock

```diff
@@ -61,21 +61,21 @@ GEM
     minitest (5.25.5)
     net-http (0.6.0)
       uri
-    nokogiri (1.18.8-aarch64-linux-gnu)
+    nokogiri (1.18.10-aarch64-linux-gnu)
```

Comment:
Do we really need all those architectures? Can't we limit it somehow?

Reply:
For sure we don't need all of them.

Reply:
This installs only the gem of the matching architecture. See https://nokogiri.org/#supported-platforms.

Take a look inside your ansible-obs container and check the gems you have installed with: `ls -l vendor/bundle/ruby/3.4.0/gems/nokogiri-*`. You'll only find the ones matching your architecture.